### PR TITLE
Fix UI of dokuwiki (hostname and first configuration only)

### PR DESCRIPTION
**Pull request by stephdl**
This pull request fix the hostname, we must require a true FQDN (mainly for lets encrypt)

https://community.nethserver.org/t/ns8-dokuwiki-wiki-fqdn-validation/23114

This PR fix the first configuration only issue of dokuwiki, the container reads the env variables just once at the first boot of the container and write the configuration files, even if you modify the environment variables,they are not not used

The fix is to display a banner to inform what to do and disable all the fields

image

https://community.nethserver.org/t/ns8-dokuwiki-instance-settings-applied-only-the-first-time/23104

refs: https://github.com/NethServer/dev/issues/6894

### CODE REVIEW

**Review by andre8244 — CHANGES_REQUESTED** [review]
- "This instance cannot be configured anymore here" -> Not true, FQDN can still be configured here. I suggest: title="Note", description="You can configure only Dokuwiki FQDN in this page. To configure other setting go to [link]" (do we know the specific link?)

**Comment by stephdl** [comment]
hum yes we could display a link, it is the FQDN of the dokuwiki instance

File: imageroot/actions/configure-module/validate-input.json [hunk 1]
```diff
@@ -49,7 +49,8 @@
     "host": {
       "type": "string",
       "description": "Host name for the wiki, like 'wiki.nethserver.org'",
-      "format": "idn-hostname"
+      "format": "hostname",
+      "pattern": "\\."
```
Comment (andre8244) [inline imageroot/actions/configure-module/validate-input.json:53]: What is this i18n string for?

File: ui/public/i18n/en/translation.json [hunk 2]
```diff
@@ -39,6 +40,9 @@
     "disabled": "Disabled",
     "instance_configuration": "Configure {instance}",
     "configuring": "Configuring...",
+    "dokuwiki_note": "Note",
+    "must_be_configured_inside_dokuwiki": "You can configure only Dokuwiki FQDN in this page. To configure other setting go to dokuwiki",
```
Comment (andre8244) [inline ui/public/i18n/en/translation.json:44]:
```suggestion
    "must_be_configured_inside_dokuwiki": "You can configure only Dokuwiki FQDN in this page. To configure other setting go to Dokuwiki webapp",
```

File: imageroot/actions/configure-module/validate-input.json (hunk 1 above)
Comment (andre8244) [reply imageroot/actions/configure-module/validate-input.json:53]: I thought this was the translation file, never mind, I'm just losing my mind :sweat_smile:

**Review by andre8244 — APPROVED** [review]

**Pull request merged**